samples/javascript_nodejs/06.using-cards/bot.js

```diff
@@ -87,10 +85,10 @@ class RichCardsBot {
 
     /**
      * Send a Rich Card response to the user based on their choice.
-     * 
+     *
      * This method is only called when a valid prompt response is parsed from the user's response to the ChoicePrompt.
-     * @param {TurnContext} turnContext
-     * @param {DialogTurnResult} dialogTurnResult 
+     * @param {Object} turnContext
```

Comment:
Let's go back and not have generic objects here. This helps with readability even if not with intellisense.